comm/src/main/scala/coop/rchain/comm/unicast.scala

```diff
@@ -0,0 +1,99 @@
+package coop.rchain.comm
+
+import java.net.{DatagramPacket, DatagramSocket}
+import scala.util.{Failure, Success, Try}
+
+/**
+  * Implement the Comm protocol for unicast (point-to-point) datagram
+  * (UDP) sockets.
+  *
+  * Unconnected datagram sockets are simple to send and receive
+  * through, though they do not guarantee delivery (even by detecting
+  * delivery failure). They are also more difficult to combine use in
+  * request-response protocols. Datagrams are limited in length and
+  * require a framing protocol at the application (this) level to
+  * recover correctly sized data.
+  *
+  * An instance of UnicastComm requires that its own location
+  * (`local`) be given; this supplies the source data for datagrams it
+  * sends.
+  */
+case class UnicastComm(val local: PeerNode) extends Comm {
```

Comment:
Darn. I had hoped that Scala freed us from that one-object-per-file business.

Reply:
It's actually fine to have multi-unit files (like you did in `comm.scala`).
I mentioned it here because I only see `UnicastComm` in `unicast.scala`. When you have just one unit in a file, you should name the file after the unit. If you plan to include other units here, it is of course fine to just keep it like this.